homeassistant/components/firmata/strings.json

```diff
@@ -1,7 +1,10 @@
 {
-  "config": {
-    "abort": {
-      "connection_error": "Could not connect to Firmata board during setup"
+    "config": {
+        "abort": {
+            "cannot_connect": "Cannot connect to Firmata board during setup"
+        },
+        "step": {
+            "import": {}
```

Comment:
If there's no form, we don't need step data here.

Reply:
Hassfest complains otherwise:
```
$ python3 -m script.hassfest
Validating json... done in 0.00s
Validating codeowners... done in 0.00s
Validating config_flow... done in 0.01s
Validating dependencies... done in 4.69s
Validating manifest... done in 0.04s
Validating services... done in 0.76s
Validating ssdp... done in 0.00s
Validating translations... done in 0.74s
Validating zeroconf... done in 0.00s
Validating coverage... done in 0.01s

Integrations: 940
Invalid integrations: 1

Found errors. Generating files canceled.

Integration firmata:
* [TRANSLATIONS] Invalid strings.json: required key not provided @ data['config']['step']. Got None
```

Reply:
Did you try with an empty object under `"step"`?

Reply:
Ah, that did the trick. Thanks!